crates/sui-storage/src/lock_service.rs

```diff
@@ -60,7 +63,7 @@ enum LockServiceCommands {
     },
 }
 
-type SuiLockResult = Result<Option<Option<TransactionDigest>>, SuiError>;
+type SuiLockResult = Result<Option<Option<LockInfo>>, SuiError>;
```

Comment:
Instead of the `LockInfo` being inside an Option, I wonder if the `None` state should really contain an Epoch.  That way we know that an object is being created for a particular epoch, would that be useful?

Reply:
I don't see how it can be used?